### PR TITLE
Added transfer_retrieve

### PR DESCRIPTION
**Pull request by jernsting**

### CODE REVIEW

File: datalad_dataverse/remote.py [hunk 1]
```diff
@@ -59,8 +59,36 @@ def transfer_store(self, key, local_file):
         resp.raise_for_status()
 
     def transfer_retrieve(self, key, file):
-        raise
-        pass
+        api = NativeApi(base_url=self.annex.getconfig('url'),
+                        api_token=os.environ["DATAVERSE_API_TOKEN"])
+        data_api = DataAccessApi(base_url=self.annes.getconfig('url'),
```
Comment (mih) [inline datalad_dataverse/remote.py:64]:
```suggestion
        data_api = DataAccessApi(base_url=self.annex.getconfig('url'),
```

**Review by mih — COMMENTED** [review]
thx, but I cannot get it to work yet

```
[2022-06-17 10:07:17.278800223] (Remote.External)   File "/home/mih/env/datalad-dev/lib/python3.9/site-packages/pyDataverse/api.py", line 132, in get_request
[2022-06-17 10:07:17.278822217] (Annex.ExternalAddonProcess) /home/mih/env/datalad-dev/bin/git-annex-remote-dataverse[1] --> DEBUG     raise OperationFailedError(
[2022-06-17 10:07:17.278847628] (Remote.External)     raise OperationFailedError(
[2022-06-17 10:07:17.278875181] (Annex.ExternalAddonProcess) /home/mih/env/datalad-dev/bin/git-annex-remote-dataverse[1] --> DEBUG pyDataverse.exceptions.OperationFailedError: ERROR: GET HTTP 404 - http://localhost:8080/api/access/datafile/MD5E-s4--ba1f2511fc30423bdbb183fe33f3dd0f.txt. MSG: {"status":"ERROR","code":404,"message":"API endpoint does not exist on this server. Please check your code for typos, or consult our API guide at http://guides.dataverse.org.","requestUrl":"http://localhost:8080/api/v1/access/datafile/MD5E-s4--ba1f2511fc30423bdbb183fe33f3dd0f.txt?User-Agent=pydataverse&key=5367f732-36bd-46ed-975a-9eeaeb98fe74","requestMethod":"GET"}
[2022-06-17 10:07:17.278907394] (Remote.External) pyDataverse.exceptions.OperationFailedError: ERROR: GET HTTP 404 - http://localhost:8080/api/access/datafile/MD5E-s4--ba1f2511fc30423bdbb183fe33f3dd0f.txt. MSG: {"status":"ERROR","code":404,"message":"API endpoint does not exist on this server. Please check your code for typos, or consult our API guide at http://guides.dataverse.org.","requestUrl":"http://localhost:8080/api/v1/access/datafile/MD5E-s4--ba1f2511fc30423bdbb183fe33f3dd0f.txt?User-Agent=pydataverse&key=5367f732-36bd-46ed-975a-9eeaeb98fe74","requestMethod":"GET"}
[2022-06-17 10:07:17.278940869] (Annex.ExternalAddonProcess) /home/mih/env/datalad-dev/bin/git-annex-remote-dataverse[1] --> ERROR ERROR: GET HTTP 404 - http://localhost:8080/api/access/datafile/MD5E-s4--ba1f2511fc30423bdbb183fe33f3dd0f.txt. MSG: {"status":"ERROR","code":404,"message":"API endpoint does not exist on this server. Please check your code for typos, or consult our API guide at http://guides.dataverse.org.","requestUrl":"http://localhost:8080/api/v1/access/datafile/MD5E-s4--ba1f2511fc30423bdbb183fe33f3dd0f.txt?User-Agent=pydataverse&key=5367f732-36bd-46ed-975a-9eeaeb98fe74","requestMethod":"GET"}

  failed to download file from remote

```

File: datalad_dataverse/remote.py (hunk 1 above)
Comment (jernsting) [reply datalad_dataverse/remote.py]: Fixed :)

**Comment by mih** [comment]
Works now!

**Review by mih — APPROVED** [review]
Cool, thx! Let's merge and optimize next.

**Pull request merged**